demos/link-prediction/metapath2vec-link-prediction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>Our other demos don't list the dataset, because typically a user doesn't care about the dataset of these "learn how to use an algorithm" demos. i.e. just "Link prediction with Metapath2Vec" would be fine IMO.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>